website/content/docs/upgrading/compatibility.mdx

```diff
@@ -30,14 +30,17 @@ at a time, one version at a time.
 
 For more details on the specifics of upgrading, see the [upgrading page](/docs/upgrading).
 
-## Protocol Compatibility Table
+## XDS Protocol Compatibility Table
```

Comment:
I _think_ this table is showing Consul agent protocol compatibility, and has nothing to do with Envoy xDS. I'm flagging this for now while I seek clarification from engineering. Regardless, I agree that what "protocol" we're referring to here needs clarification!

Reply:
This refers to the RPC protocol version.

https://github.com/hashicorp/consul/blob/b29e5894f2322c9294b1f03c9b264ae2e6ae0eb0/agent/config/config.go#L212

I believe support for xDS v2 was dropped in hashicorp/consul#11354 when that PR dropped support for Envoy 1.16.5 in [Consul 1.11.0](https://github.com/hashicorp/consul/blob/main/CHANGELOG.md#1110-december-14-2021). At the time, all of the more recent, supported versions of Envoy also the newer xDS v3.